packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -22,6 +22,9 @@ export function perStepLocalTimeout(): number {
 /** Selector string to get element by data-ui-id property */
 export const dataUiId = (id: string): string => `[data-ui-id="${id}"]`;
```

Comment:
If we're moving to data-testid, should we mark this as deprecated at least?

Reply:
I posted it in a common chat and I will push an another PR if everyone is okay with the change